courses/machine_learning/deepdive2/how_google_does_ml/solutions/analyze_with_bigquery_solution.ipynb

```diff
@@ -37,7 +37,7 @@
   },
```

Comment:
<p>It would be good to be able to see all the columns. Maybe there is a %bigquery parameter to be set to avoid column truncation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>All of the columns are present.  There are 32 columns total, you must scroll to see them all.  I cannot think of a way to work around that aspect of the output.  </p>

Reply:
<p>All columns are in fact present, you simply have to scroll to see them all.   There is no BigQuery option to reduce the width of the 32 columns.</p><p> </p><p>I will also update the SQL syntax format to be consistent with the other format fixes.</p>

Reply:
<p>You can do it with pandas (pushed a commit in the notebook directly):</p><p> </p><p>import pandas as pd</p><p>pd.options.display.max_columns = 50</p>